packages/commitlint-config/index.spec.js

```diff
@@ -69,4 +70,5 @@ describe('Commitlint Config Cozy', () => {
       }
     })
   })
+  */
```

Comment:
I would suggest deleting the code block instead of commenting on it because we never know why it is commented on.